app.js

```diff
@@ -139,4 +150,108 @@ async function getAgendaitemsRequestUrls(agendaId) {
   return urls;
 }
 
+async function warmupConcepts() {
+  const urls = [
+    await getConceptsHasNoNarrowerBatchedRequestsUrls(CONCEPT_SCHEMES.MEETING_TYPE),
```

Comment:
Just for consistency/readability, can you also use `Promise.all` with a `map` on `[CONCEPT_SCHEMES.MEETING_TYPE]` here?